meson_options.txt

```diff
@@ -10,6 +10,8 @@ option('chafa', type: 'feature', value: 'auto',
 option('icu', type: 'feature', value: 'auto',
   description: 'Enable ICU library unicode functions')
 option('graphite', type: 'feature', value: 'disabled',
+  description: 'Deprecated use graphite2 option insatead')
```

Comment:
Typo `instead`.

Can we hide this one?

Reply:
> Can we hide this one?

I don’t think so, that probably what the meson PR would do.